### DISCUSSION TITLE
Dynamic (fading) stroke size along a bezier path

### DISCUSSION THREAD

**Discussion by BenevolentCheese** [ImageMagick/ImageMagick#4925] [Help]
I'm looking to draw along a bezier path with a stroke that starts at full size and gradually shrinks to zero by the end of the path. This would approximate photoshop's "shape dynamics -> size jitter: fade" option. Here's an example from photoshop: https://i.imgur.com/bY1a3CQ.jpg

I've looked around and it doesn't look like this is possible with a simple option. Is that correct? If that's so, does anyone have ideas for how I might execute this? My only idea at the moment is constructing a polygon out of the desired shape and then using IM to draw that. But that's, like, a lot. I'd obviously prefer an easier option!

**Comment by BenevolentCheese** [comment]
I guess another option is to draw many circles along the path at gradually smaller sizes but I'd wager that is very slow.

**Comment by snibgo** [comment]
What is the starting position? If you have a curve that defines the centre line, than you could duplicate that twice, rotate one slightly clockwise around the tail, rotate the other anticlockwise, reverse one of them, and join them at the head. That gives a closed curve.

Another possibility: make the basic shape as a raster, and distort it however you want.
